central/main.go

```diff
@@ -864,6 +866,12 @@ func customRoutes() (customRoutes []routes.CustomRoute) {
 			ServerHandler: certHandler.BackupCerts(listener.Singleton()),
 			Compression:   true,
 		},
+		{
+			Route:         "/metrics",
```

Comment:
We need a comment why we need a second metrics endpoint and what's the diffecerence between the old one.
cc: @stehessel